index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>VideoPoker</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1 id="header">Video Poker</h1>
+  <!-- Import program logic -->
+  <script src="constants.js"></script>
+  <script src="script.js"></script>
+<!-- <div class="game">
+  <div class="game-play">
+    <div class="buttons">
+      <button>Start or Draw/Deal or Deal</button>
+    </div>
+    <div class="dealt-cards">
+      <div class="single-card">
+        <div class="name default-fill">**</div>
+        <div class="suit default-fill">**</div>
+      </div>
+      <div class="single-card">
+        <div class="name red">A</div>
+        <div class="suit red">💗</div>
+      </div>
+      <div class="single-card">
+        <div class="name black">A</div>
+        <div class="suit black">💗</div>
+      </div>
+      <div class="single-card">
+        <div class="name black">A</div>
+        <div class="suit black">💗</div>
+      </div>
+    </div>
+    <div class="status">
+      Display the status of the last game
+    </div>
+  </div >
+
+  <div class="pay-table">
+    <table>Pay Table
+      <tr>
+        <th>Game Type</th>
+        <th>Points</th>
+      </tr>
+      <tr>
+        <td>Five of a kind</td>
+        <td>2</td>
+      </tr>
+    </table>
+  </div>
+</div>
+
+<div class="result">
+  <div>
+  <button>Show / Hide Score</button>
+</div>
+  <div><table class="result-table">Game Result
+    <tr>
+      <th>Game Count</th>
+      <th>Result</th>
+      <th>Current Score</th>
+      <th>Final Set Of Cards Played</th>
+    </tr>
+    <tr>
+      <td>1</td>
+      <td>win</td>
+      <td>101</td>
+      <td>
+        <div class="single-card reduced-card">
+          <div class="name red reduced-size">A</div>
+          <div class="suit red reduced-size">💗</div>
+        </div>
+        <div class="single-card reduced-card">
+          <div class="name red reduced-size">A</div>
+          <div class="suit red reduced-size">💗</div>
+        </div>
+      </td>
+    </tr>
+  </table>
+</div>
+</div> -->
```

Comment:
Please remove commented-out code before submitting, else add a comment above explaining why you left this commented-out code here. Otherwise it's like leaving rubbish around the house.

Reply:
Removed all the commented out code. That code was submitted, because I was still working on the css portion and not sure whether I might need it or not.
Will take care in future.